### ISSUE TITLE
No export member 'TWrapMiddleware'

### ISSUE THREAD

**Issue by jacorall** [nhttp/nhttp#5]
Just installed deno (version 1.13.0) on a new VM, made a test folder, created an app.ts file and copied the first example from 'Usage' section in the Nhttp documentation to it. When I run it with the following command:
```
deno run --unstable --allow-net --allow-read app.ts
```
I get the following error:
```
error: TS2305 [ERROR]: Module '"https://deno.land/x/nhttp@0.8.5/src/types.ts"' has no exported member 'TWrapMiddleware'.
```
Please let me know if I'm doing something wrong.

**Comment by herudi** [comment]
Thanks for created this issue... i will fix ASAP in the next release...

**Comment by herudi** [comment]
fix in new version [1.0.0](https://github.com/nhttp/nhttp/releases/tag/1.0.0)
```ts
import { NHttp } from "https://deno.land/x/nhttp@1.0.0/mod.ts";
```

**Comment by jacorall** [comment]
Thank you for the quick response.
